README.md

```diff
@@ -213,20 +214,22 @@ collections:
   - name: redhat.trusted_profile_analyzer
 ```
 
-or with the classic ansible command
+Or by using the following Ansible commands:
 
 ```shell
 ANSIBLE_ROLES_PATH="roles/" ansible-playbook -i inventory.ini play.yml -vv
 ```
 
-Note that if you install any collections from Ansible Galaxy, they will not be upgraded automatically when you upgrade the Ansible package.
+> [!NOTE]
+If you install any collection from Ansible Galaxy, upgrading the Ansible package is not automatically done.
 To upgrade the collection to the latest available version, run the following command:
 
 ```
 ansible-galaxy collection install redhat.trusted_profile_analyzer --upgrade
 ```
 
-You can also install a specific version of the collection, for example, if you need to downgrade when something is broken in the latest version (please report an issue in this repository). Use the following syntax to install version 0.2.0:
+You can also install a specific version of the collection.
+You can also install a specific version of the collection. (please report an issue in this repository). Use the following syntax to install version 0.2.0:
```

Comment:
Please remove this line, it's just a duplicate of 231, with the added parenthetical statement, which we don't want in there.

Reply:
The suggestion is this:

You can also install a specific version of the collection.
For example, if you need to downgrade when something is broken in the latest version.

Reply:
fixed